Scripts/UI/SabreCSGPreferences.cs

```diff
@@ -116,6 +116,14 @@ public static void PreferencesGUI()
                 SceneView.RepaintAll();
             }
 
+            EditorGUILayout.Space();
+            EditorGUI.indentLevel = 1;
```

Comment:
Modifying indent level directly like this is probably bad practice, I'd suggest instead using [IndentLevelScope](https://docs.unity3d.com/ScriptReference/EditorGUI.IndentLevelScope.html)